Description: Disable transitive linking
Author: Paul Klos <kolab@klos2day.nl>
Last-Update: 2013-07-07
Bug: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=714102

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -22,6 +22,7 @@
     ${SCHEMA_SOURCEFILES}
 )
 add_dependencies(kolabxml generate_bindings)
+set_target_properties(kolabxml PROPERTIES LINK_INTERFACE_LIBRARIES "")
 target_link_libraries(kolabxml ${XERCES_C} ${Boost_LIBRARIES} ${UUID} ${CURL})
 
 # For the core library we can be stricter when compiling. This doesn't work with the auto generated code though.
