Index: libkolabxml-0.7.0/CMakeLists.txt
===================================================================
--- libkolabxml-0.7.0.orig/CMakeLists.txt	2012-07-19 11:23:24.000000000 +0100
+++ libkolabxml-0.7.0/CMakeLists.txt	2012-07-19 12:45:30.410964470 +0100
@@ -76,7 +76,7 @@
 
 set(LIB_INSTALL_DIR lib CACHE STRING "The directories where to install libraries to")
 set(INCLUDE_INSTALL_DIR include CACHE STRING "The directory where to install headers to")
-set(INCLUDE_INSTALL_DIR ${INCLUDE_INSTALL_DIR}/kolabxml)
+set(INCLUDE_INSTALL_DIR ${INCLUDE_INSTALL_DIR}/kolab)
 set(CMAKECONFIG_INSTALL_DIR ${LIB_INSTALL_DIR}/cmake/Libkolabxml )
 
 configure_file(libkolabxml-version.h.cmake "${CMAKE_BINARY_DIR}/libkolabxml-version.h" @ONLY)
