diff --git a/src/php/CMakeLists.txt b/src/php/CMakeLists.txt
index aec9cd6..267bb81 100644
--- a/src/php/CMakeLists.txt
+++ b/src/php/CMakeLists.txt
@@ -20,10 +20,12 @@ if (APPLE)
     set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -flat_namespace -undefined suppress" )
 endif()
 
-find_package(PHP4 5.3 REQUIRED)
+if (NOT PHP4_INCLUDE_PATH OR NOT PHP4_EXECUTABLE)
+    find_package(PHP4 5.3 REQUIRED)
+endif (NOT PHP4_INCLUDE_PATH OR NOT PHP4_EXECUTABLE)
 
-if (PHP4_FOUND)
-    include_directories(${PHP4_INCLUDE_PATH})
+if (PHP4_FOUND OR (PHP4_INCLUDE_PATH AND PHP4_EXECUTABLE))
+    include_directories(${PHP4_INCLUDE_PATH} "${PHP4_INCLUDE_PATH}/Zend" "${PHP4_INCLUDE_PATH}/TSRM" "${PHP4_INCLUDE_PATH}/main")
     add_library(phpbindings SHARED ${KOLAB_SWIG_PHP_SOURCE_FILE})
     target_link_libraries(phpbindings kolabxml)
     SET_TARGET_PROPERTIES(phpbindings PROPERTIES OUTPUT_NAME "kolabformat")
@@ -39,6 +41,6 @@ if (PHP4_FOUND)
             ${CMAKE_CURRENT_BINARY_DIR}/kolabformat.php
             DESTINATION ${PHP_INSTALL_DIR}
         )
-else(PHP4_FOUND)
+else(PHP4_FOUND OR (PHP4_INCLUDE_PATH AND PHP4_EXECUTABLE))
     message(WARNING "not building php bindings because php was not found")
-endif (PHP4_FOUND)
+endif (PHP4_FOUND OR (PHP4_INCLUDE_PATH AND PHP4_EXECUTABLE))
